pom.xml

```diff
@@ -93,6 +98,27 @@
           </arguments>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>net.alchim31.maven</groupId>
+        <artifactId>scala-maven-plugin</artifactId>
+        <executions>
+          <execution>
```

Comment:
This fails on my computer

Reply:
I didnt get any error. send me error stack